extensions/eventtap/init.lua

```diff
@@ -177,29 +177,41 @@ function module.rightClick(point, delay)
     module.event.newMouseEvent(module.event.types["rightMouseUp"], point):post()
 end
 
---- hs.eventtap.middleClick(point[, delay])
```

Comment:
if you leave the button number off of `hs.eventtap.otherClick`, it defaults to 2, which is the middle click.